tests/model_monitoring/test_applications/test_app_context.py

```diff
@@ -30,9 +30,13 @@
 @pytest.mark.parametrize("method", ["log_artifact", "log_dataset", "log_model"])
 def test_log_object_signature(method: str) -> None:
     """Future-proof the `log_x` method of MM app context with respect to the project object"""
-    assert inspect.signature(
+    monitoring_parameters = inspect.signature(
         getattr(MonitoringApplicationContext, method)
-    ) == inspect.signature(getattr(MlrunProject, method))
+    ).parameters.values()
+    project_parameters = inspect.signature(
+        getattr(MlrunProject, method)
+    ).parameters.values()
+    assert all(item in monitoring_parameters for item in project_parameters)
```

Comment:
I think that the following will work:
```suggestion
    assert project_parameters <= monitoring_parameters
```

Reply:
Nope because `Parameter` is an object and they didn't implement  a <= operator